templates/definition/charger/daheimladen.yaml

```diff
@@ -0,0 +1,18 @@
+template: daheimladen
+description: Daheimladen (Cloud API)
+requirements:
+  sponsorship: true
+params:
+- name: token
+  help:
+    de: "Sie bekommen dieses Access Token vom Daheimladen Support: info@daheimladen.de"
+    en: "Request your access token from Daheimladen Support: info@daheimladen.de"
+- name: stationid
+  example: "xxxxxxxxxxxx990"
+  help:
+    de: "Software Serial Number displayed on the display of the wallbox"
```

Comment:
Sprachen vertauscht. Einfacher:

    Serial Number displayed on the wallbox

Was ist eine "software serial number"?

Reply:
ups. Formulierung habe ich aus dem PR: https://github.com/evcc-io/evcc/pull/1699#issuecomment-980637761

Reply:
Wenn du eine bessere Formulierung hast, her damit. Ich kenne die Wallbox ja nicht ;)